packages/mural-picker/src/components/mural-picker/index.tsx

```diff
@@ -134,13 +144,20 @@ export default class MuralPicker extends React.Component<
     });
   };
 
+  // Abort in-flight requests and reset the ApiClient instance to prepare for
+  // new requests
+  private abortRequests = () => {
+    this.apiClient.abort();
+    this.apiClient = this.apiClient.clone();
+  };
```

Comment:
**question**: Do we _actually_ need to do this?

Reply:
Thanks. As it turns out, no we actually don't!